statsmodels/tsa/tests/test_arima_process.py

```diff
@@ -59,8 +60,7 @@ def test_arma_acovf_persistent():
     corrs = .9995**np.arange(10)
     expected = sig2*corrs
     assert_equal(res.ndim, 1)
-    assert_allclose(res, expected, atol=1e-6)
-    # atol=7 breaks at .999, worked at .995
+    assert_allclose(res, expected)
```

Comment:
Does the change in this PR improve the match?  (i.e. the resolve the comment above)

Reply:
Yes, all of the precision improvements are due to the analytic compuation.